README.md

```diff
@@ -30,26 +30,26 @@ npm install -g clever-tools@beta
 
 ##### Warning
 
-Bintray is ending on May 1st 2021, we will migrate hosting as soon as possible. Please refer to the dedicated [issue](https://github.com/CleverCloud/clever-tools/issues/454) to track the migration.
+We recently moved from Bintray to a self hosted Nexus repository available at [https://nexus.clever-cloud.com](https://nexus.clever-cloud.com). You have to update the URL of your repository with the new one. If you experience any issue please let us know on the following [issue https://github.com/CleverCloud/clever-tools/issues/454](https://github.com/CleverCloud/clever-tools/issues/454).
 
 ---
 
 If you are using a GNU/Linux distribution that uses `.deb` packages like Debian or Ubuntu, you can run:
 
 ```sh
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "379CE192D401AB61"
-echo "deb https://dl.bintray.com/clevercloud/deb stable main" | tee -a /etc/apt/sources.list
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "a7b8f38c536f1df2"
+echo "deb https://nexus.clever-cloud.com/repository/deb bionic stable main" | tee -a /etc/apt/sources.list
```

Comment:
Does not work. At the moment the correct repository configuration seems to be:
```suggestion
echo "deb [arch=amd64] https://nexus.clever-cloud.com/repository/deb bionic main" | tee -a /etc/apt/sources.list
```

Reply:
I just tested on `ubuntu:latest` docker image and it works with this configuration:
```
deb [arch=amd64] https://nexus.clever-cloud.com/repository/deb stable main
```

Reply:
I confirm. The suggested change above has now been updated.